app/main/posts/views/share/post-export.directive.js

```diff
@@ -90,64 +45,12 @@ function PostExportController(
         if (!$scope.filters || _.isEmpty($scope.filters)) {
             $scope.filters = PostFilters.getDefaults();
         }
-        var format = 'csv';  //@todo handle more formats
         // Prepare filters for export
-        var query = angular.extend({}, PostFilters.getQueryParams($scope.filters), {
-            format: format
-        });
+        var entity_type = 'post';
```

Comment:
Why is this hard-coded? Just curious.

Reply:
Yes, maybe we can do this in the endpoint instead...